packages/xo-web/src/xo-app/new-vm/index.js

```diff
@@ -501,6 +504,8 @@ export default class NewVm extends BaseComponent {
       tags: state.tags,
       vgpuType: get(() => state.vgpuType.id),
       gpuGroup: get(() => state.vgpuType.gpuGroup),
+      hvmBootFirmware:
+        state.hvmBootFirmware === '' ? null : state.hvmBootFirmware,
```

Comment:
What is `null` and why is it only handled here?

Reply:
default value is represented by an empty string. Therefore when `hvmBootFirmware: null` is sent to the server, nothing will happen and `hvmBootFirmware` will not be set.